tachyon/zk/plonk/circuit/assignment.h

```diff
@@ -105,12 +105,14 @@ class Assignment {
 
   // Creates a new (sub)namespace and enters into it.
   //
+  // TODO(chokobole): Update comment since NamespacedLayouter comes in.
```

Comment:
I don't get the comment. Did you mean
```suggestion
  // TODO(chokobole): Update comment when NamespacedLayouter comes in.
```
? To remind in the future?
If my guess is correct, could you please fix all the comments below as well?

Reply:
Oh yeap you're correct! I'll fix them al.